src/components/Title.jsx

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import { PiStarFourFill } from "react-icons/pi";
 
-const Title = ({ text }) => {
+const Title = ({ text, className }) => {
```

Comment:
```suggestion
const Title = ({ text, size = "text-3xl" }) => {
```

rename it to size instead. you can also put a default value for text size.